### PR TITLE
Fix sample prediction break

### PR DESCRIPTION
**Pull request by Laurae2**
Fixing the following reported issue: predict.cubist unable to predict properly using sample (cubistControl) #1

tl;dr explanation: when using sample parameter in cubistControl, predictions are breaking down immediately (unable to predict properly, non-sensical predictions with large error, near zero correlation, etc.). This fix solves this issue without creating other issues.

Extra explanation about the fix:

There is probably a proper way to do it using a better regex but this version works perfectly, whether there is sample defined or not. Init has no impact overall.

It removes everything starting from "sample" and before "entries".

I found no impact removing redn (it shows up between sample and entries when using commitees).
redn = final error / (sum of errors / (number of commitees - 1)), just a calculated output value (it happens to be read when fed into Cubist into ErrReduction variable, but is not used at all to predict).

### CODE REVIEW

**Comment by topepo** [comment]
Yeah, this kinda fell through the cracks for a while. Thakns
